python_tests/pytest_ops.py

```diff
@@ -131,23 +132,17 @@ def schedule(self):
     except Exception as e:
         exception = e
 
-    assert exception is not None, "Expected an exception"
-    exception_str = "Attempting to add to a completed definition!"
-    assert exception_str in str(
-        exception
-    ), "Failed to find correct expection error message"
+    pytest.raises(TypeError, "Attempting to add to a completed definition")
```

Comment:
wasn't this supposed to replace the `try/except` pattern?
I thought the example in the doc you linked suggests something like:
```
with pytest.raise(TypeError, "..."):
    fd = SchedError()
    fd.execute(...)
```

Reply:
NVM, I saw that this pattern is used below. Then why are we calling the pytest.raises here? and why are we catching exception here, wasn't it supposed to be caught by the pytest.raise?

Reply:
Good catch! I must have missed changing the `definition_op_in_schedule_error_test_fn`  function.